mtp_cashbook/templates/faq.html

```diff
@@ -0,0 +1,81 @@
+{% extends 'base.html' %}
+{% load i18n %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+{% block body_classes %}{{ block.super }} mtp-credits{% endblock %}
+
+{% block content %}
+  {{ block.super }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <a class="govuk-back-link" href="{% url 'home' %}">
+        {% trans 'Back' %}
+      </a>
+
+      <header>
+        <h1 class="govuk-heading-xl">{{ view.title }}</h1>
+      </header>
+
+
+      <h2 class="govuk-heading-m" id="faq-account-requested">
+        {% trans "I requested an account, but heard nothing" %}
+      </h2>
+
+      <p class="govuk-body">
+        {% blocktrans trimmed %}
+          Either remind your business hub manager to accept your request, or <a href="{{ sign_up_url }}" class="govuk-link">request an account</a> again.
```

Comment:
you don't need `govuk-link` on links almost anywhere (only when it's side-by-side with a button or not inside a paragraph/heading)